.github/workflows/virtual-integration.yml

```diff
@@ -441,6 +441,12 @@ jobs:
         run: |
           mage publish:releaseManifest
 
+      - name: Build new installer
+        run: mage newinsaller:build
+
+      - name: Test new installer
+        run: mage newinsaller:test
+
```

Comment:
We only need to run this when the `/newinstaller` folder is chanced. We should add that check.
Also I would suggest creating a new workflow yaml instead of mixing up with `virtual-integration`